src/jest/jest.schema.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
The schema is not used or referenced in the collection.schema.
You can remove it for now.